### ISSUE TITLE
v1.39.0 plugin crashed!

### ISSUE THREAD

**Issue by PiotrPardo** [MyPureCloud/terraform-provider-genesyscloud#1078]
Hello,

Today after provider update I received a lot of errors.

With resources to update/add I received this
```
│ The plugin.(*GRPCProvider).ValidateResourceConfig request was cancelled.
│ Error: Request cancelled
```

```
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may
│ contain more details.
│ Error: Plugin did not respond
```

Full details logs

```
Stack trace from the terraform-provider-genesyscloud_v1.39.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=]

goroutine 691 [running]:
terraform-provider-genesyscloud/genesyscloud/architect_schedules.createArchitectSchedules({})
	terraform-provider-genesyscloud/genesyscloud/architect_schedules/resource_genesyscloud_architect_schedules.go:91 +0x77b
terraform-provider-genesyscloud/genesyscloud/provider.runWithPooledClient.func1({}, 0x0?, {})
	terraform-provider-genesyscloud/genesyscloud/provider/sdk_client_pool.go:134 +0x283
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(, {})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:806 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(, {,},,, {,})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/resource.go:937 +0xa7e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(, {?,?},)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.34.0/helper/schema/grpc_provider.go:1153 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(, {?,?},)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:865 +0x3dc
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({?,}, {,},, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:518 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(, {,}, {,},,,, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xe69
google.golang.org/grpc.(*Server).handleStream(, {,},)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0x106f
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x91
created by google.golang.org/grpc.(*Server).serveStreams.func2
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x145

Error: The terraform-provider-genesyscloud_v1.39.0 plugin crashed!
```

**Comment by charliecon** [comment]
Hi @PiotrPardo 

We're working on a fix right now which will be released on v1.40.0 in the next day or two.

Thanks for reporting!

-Charlie

**Comment by charliecon** [comment]
Fixed on v1.40.0
